cosmos-gui/test/node/mysql_driver_test.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-tidoop (FI-WARE project).
+ *
+ * fiware-tidoop is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-tidoop is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-tidoop. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with
+ * francisco dot romerobueno at telefonica dot com
+ */
+
+/**
+ * mysql_driver tests
+ *
+ * Author: frb
+ */
+
+// Module dependencies
+var assert = require('assert');
+var rewire = require('rewire');
+var mysqlDriver = rewire('../../src/mysql_driver.js');
+
+// Testing variables
+var idm_username = 'frb@tid.es';
+var username = 'frb1';
+var password = '12345';
+var hdfsQuota = 5;
+
+// Mock the mysql connection
+var connectionMock = {
+    connect: function(callback) {
+        return callback(null);
+    },
+    query: function(query, values, callback) {
+        query = query || '';
+        values = values || [];
+        callback = callback || function(error, result) {};
```

Comment:
He, he :) The other option is to check `if (callback)` and only calling it if it exists :) Not to create an empty function :) What do you think?